src/xma/include/app/xmahw.h

```diff
@@ -43,6 +45,7 @@ typedef enum {
   XMA_CMD_STATE_PSK_CRASHED = 7, //PS kernel has crashed
   XMA_CMD_STATE_MAX = 8 // Always the last one
 } XmaCmdState;
+static_assert(sizeof(XmaCmdState) <= 4, "XmaCmdState size must be less than 5 bytes");
```

Comment:
`static_assert(sizeof(XmaCmdState) <= sizeof(int32_t), "XmaCmdState size must be less than 4 bytes");`

Reply:
Move the static_assert to a .cpp file.

Reply:
Done